recipes/configs/llama3_2_vision/90B_generation_distributed.yaml

```diff
@@ -0,0 +1,51 @@
+# Config for running the InferenceRecipe in dev/generate_v2.py to generate output
+# from a Llama3.2 90B Vision Instruct model
+#
+# This config assumes that you've run the following command before launching:
+#   tune download meta-llama/Llama-3.2-90B-Vision-Instruct --output-dir /tmp/Llama-3.2-90B-Vision-Instruct --ignore-patterns "original/consolidated*"
+#
+# To launch, run the following command from root torchtune directory:
+#    tune run --nproc_per_node 8 dev/generate_v2_distributed --config llama3_2_vision/90B_generation_distributed
+
+output_dir: ./
+
+# Model arguments
+model:
+  _component_: torchtune.models.llama3_2_vision.llama3_2_vision_90b
+
+parallelize_plan:
+  _component_: torchtune.models.llama3_2_vision.llama3_2_vision_tp_plan
```

Comment:
I know it's due to the wildcard issue but it's confusing to expose this in the config and then use `parallelize_llama3_2_vision` in the recipe. We should simplify this (personally I would just rip out the wildcards until they're actually supported, but up to you)